infra/feast-operator/config/manager/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+resources:
+- manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: feastdev/feast-operator
+  newTag: 0.40.0
```

Comment:
is this the latest feast release? if so it should be https://github.com/feast-dev/feast/releases/tag/v0.40.1

Reply:
```suggestion
  newTag: 0.40.1
```